src/api/Notifications.ts

```diff
@@ -55,7 +49,7 @@ export default class Notifications extends API {
     `https://notifications-processor.decentraland.zone`
   )
   static Token = env("NOTIFICATION_SERVICE_TOKEN", "")
-  static ExplorerURL = env("EXPLORER_URL", "https://play.decentraland.org/")
+  static ExplorerURL = env("DECENTRALAND_URL", "https://decentraland.org/jump/")
```

Comment:
Also, if this is not explorer url anymore. Would you mind updating the constant name here? 
```suggestion
  static JumpInSiteURL = env("JUMP_IN_SITE_URL", "https://decentraland.org/jump/")
```

Reply:
One last thing. Since the notification is about events, take in consideration that the jump in needs to go to the events jump in site. Something like https://decentraland.org/jump/events/?position=-140,-86 or like https://decentraland.org/jump/events/?id=025e7f18-f13e-40b7-ad9c-e826688f3d0e